### PR TITLE
add 4 new citations for March 2021

### PR DESCRIPTION
**Pull request by ljhwang**
Adding 4 new publications for March 2021

### CODE REVIEW

**Review by MFraters — APPROVED** [review]
Looks good to me. Could you squash the commits into one?

**Comment by anne-glerum** [comment]
Isn't it three times the same entry?

**Comment by ljhwang** [comment]
Yikes, you are right.  And I thought I checked this thoroughly!

File: doc/manual/citing_aspect.bib [hunk 1]
```diff
@@ -891,4 +891,68 @@ @article {BredowM56-2020-2
...
+    issn = {0956-540X},
+    doi = {10.1093/gji/ggab051},
+    url = {https://doi.org/10.1093/gji/ggab051},
+    eprint = {https://academic.oup.com/gji/article-pdf/225/3/1624/36583045/ggab051.pdf},
+}
+
+@article{https://doi.org/10.1029/2019JB018726,
+author = {Heyn, Björn H. and Conrad, Clinton P. and Trønnes, Reidar G.},
```
Comment (bobmyhill) [inline doc/manual/citing_aspect.bib:913]: Are we allowing special characters now? They are normally escaped in bibtex:
Bj{\"{o}}rn and Tr{\o}nnes

**Comment by MFraters** [comment]
wow, I didn't notice that (mostly looked whether the entries names where correct or if there were any weird symbols) :s

Good catch Anne!

File: doc/manual/citing_aspect.bib (hunk 1 above)
Comment (ljhwang) [reply doc/manual/citing_aspect.bib:913]: thanks Bob

**Comment by ljhwang** [comment]
This is closed as this has gotten messed up.

**Pull request closed without merging**